layouts/_default/_markup/render-link.html

```diff
@@ -1 +1 @@
-<a href="{{ .Destination | safeURL }}"{{ with .Title}} title="{{ . }}"{{ end }} {{ if strings.HasPrefix .Destination "http" }} class="external-link" target="_blank" rel="noopener"{{ end }}>{{ .Text | safeHTML }}</a>
\ No newline at end of file
+<a href="{{ .Destination | safeURL }}"{{ with .Title}} title="{{ . }}"{{ end }} {{ if or (or (strings.HasPrefix (lower .Destination) "http://") (strings.HasPrefix (lower .Destination) "https://")) ( strings.HasPrefix .Destination "//" ) }} class="external-link" target="_blank" rel="noopener"{{ end }}>{{ .Text | safeHTML }}</a>
```

Comment:
I think we could simplify here using `findRE`.

Maybe something like:
```
{{ if findRE(`^(http)?s?:?//` (lower .Destination)) }}
```

Reply:
I was originally avoiding using a regex for performance reasons but now I realise that the performance impact is absolutely negligible.